guidelines/index.html

```diff
@@ -3859,7 +3859,7 @@ <h2 id="x6-glossary"><span class="secno">6. </span>用語集<span class="permali
                 <dt><dfn data-dfn-type="dfn" id="dfn-synchronized-media">同期したメディア (synchronized media)</dfn></dt>
 <dd>
    
-   <p>情報を提示するために、他のフォーマットと同期した<a href="#dfn-audio" class="internalDFN" data-link-type="dfn">音声</a>もしくは<a href="#dfn-video" class="internalDFN" data-link-type="dfn">映像</a>、又は時間に依存するインタラクティブな構成要素と同期した音声又は映像。ただし、そのメディアがメディアによるテキストの代替であって、<a href="#dfn-media-alternative-for-text" class="internalDFN" data-link-type="dfn">メディアによる代替</a>であることが明確にラベル付けされているものは除く。[mark]
+   <p>情報を提示するために、他のフォーマットと同期した<a href="#dfn-audio" class="internalDFN" data-link-type="dfn">音声</a>もしくは<a href="#dfn-video" class="internalDFN" data-link-type="dfn">映像</a>、及び／又は時間に依存するインタラクティブな構成要素と同期した音声もしくは映像。ただし、そのメディアが<a href="#dfn-media-alternative-for-text" class="internalDFN" data-link-type="dfn">メディアによるテキストの代替</a>であって、メディアによるテキストの代替であることが明確にラベル付けされているものは除く。
```

Comment:
今の訳では原文ではas suchとなっている部分を展開しているので、展開せずに以下のようにしても良いかもしれません。

```suggestion
   <p>情報を提示するために、他のフォーマットと同期した<a href="#dfn-audio" class="internalDFN" data-link-type="dfn">音声</a>もしくは<a href="#dfn-video" class="internalDFN" data-link-type="dfn">映像</a>、及び／又は時間に依存するインタラクティブな構成要素と同期した音声もしくは映像。ただし、そのメディアが<a href="#dfn-media-alternative-for-text" class="internalDFN" data-link-type="dfn">メディアによるテキストの代替</a>であって、そのように明確にラベル付けされているものは除く。
```